pkg/image/skopeo.go

```diff
@@ -120,9 +127,16 @@ var extractImageLayers = func(dest string) error {
 		layer := strings.TrimPrefix(layerID, "sha256:")
 		filePath := filepath.Join(dest, dataTmpDir, layer)
 
-		if err := util.UnArchiveLocalTar(filePath, dest, "z"); err != nil {
-			return errors.Wrap(err, "could not extract layer tar")
+		//prepend z option to the beggining of untar arguments
+		args := append([]string{"z"}, arg...)
+
+		if err := util.UnArchiveLocalTar(filePath, dest, args...); err != nil {
+			//ignore errors if specific file extract was requested - we validate wheather the file was extracted at the end of the sequenec
```

Comment:
//ignore errors if specific file extract was requested - we validate whether the file was extracted at the end of the sequence

just some small typos.